containers/zammad/docker-entrypoint.sh

```diff
@@ -74,7 +74,8 @@ if [ "$1" = 'zammad-init' ]; then
     bundle exec rake db:seed
 
     # create autowizard.json on first install
-    if [ -n "${AUTOWIZARD_JSON}" ]; then
+    if [ $((${#AUTOWIZARD_JSON} % 4)) -eq 0 ]; then
```

Comment:
We could also check for valid base64 by:
```suggestion
    if base64 -d <<< ${AUTOWIZARD_JSON} &>> /dev/null; then
```